integration/presto/src/main/java/org/apache/carbondata/presto/impl/CarbonLocalInputSplit.java

```diff
@@ -127,7 +128,8 @@ public CarbonLocalInputSplit(@JsonProperty("segmentId") String segmentId,
       @JsonProperty("deleteDeltaFiles") String[] deleteDeltaFiles,
       @JsonProperty("blockletId") String blockletId,
       @JsonProperty("detailInfo") String detailInfo,
-      @JsonProperty("fileFormatOrdinal") int fileFormatOrdinal) {
+      @JsonProperty("fileFormatOrdinal") int fileFormatOrdinal,
+      boolean isDistributedPruningEnabled) {
```

Comment:
please also keep  @JsonProperty for this

Reply:
done